app/responders/core_responder.rb

```diff
@@ -0,0 +1,18 @@
+module EventResponders
+
+    class CourseResponder < ApplicationResponder
```

Comment:
use two space indentation, move each class to its own file

Reply:
Fixed